internal/armada/repository/job.go

```diff
@@ -139,17 +139,28 @@ func (repo *RedisJobRepository) AddJobs(jobs []*api.Job) ([]*SubmitJobResult, er
 }
 
 func (repo *RedisJobRepository) RenewLease(clusterId string, jobIds []string) (renewedJobIds []string, e error) {
+	// TODO: If we can pass in the queue, we don't need to load jobs from Redis.
 	jobs, err := repo.GetExistingJobsByIds(jobIds)
 	if err != nil {
 		return nil, err
 	}
-
-	leasedJobs, err := repo.leaseJobs(clusterId, jobs)
+	jobsById := make(map[string]*api.Job)
```

Comment:
you can assign this map with the correct length

Reply:
Fixed